src/Services/ObexManager.vala

```diff
@@ -20,8 +20,10 @@ public class BluetoothIndicator.Services.ObexManager : Object {
     public signal void transfer_removed (BluetoothIndicator.Services.Obex.Transfer transfer);
     public signal void transfer_active (string address);
     private GLib.DBusObjectManagerClient object_manager;
+    public GLib.HashTable<BluetoothIndicator.Services.Obex.Transfer, string> transferact;
```

Comment:
I am in favour of using meaningful variable names where possible.  How about `active_transfers`?

Reply:
😃👍